student-work/felice-berk/API_Final_Project/POST.md

```diff
@@ -77,3 +77,12 @@ Beneath the code snippet is a table explaining the components of the POST.
 | ice        	| string    	| "yes", "none"                                 	| Choice of adding ice to the drink. Choice of none means no ice.	|
 
 
+## POST Response Example 
+
+When a correct order is placed, the server replies to the app with an acknowledgment. 
+This is not displayed to the user.  
+
+```JSON
```

Comment:
```suggestion
```HTTP
```

Reply:
The Response is an HTTP Code not JSON, so this is why the OK is in red, the compiler can't compile it. 
If HTTP still makes it red - remove the HTTP and leave it blank